tests/browser/modules/selection-state.spec.js

```diff
@@ -44,6 +44,36 @@ describe('Browser: SelectionState', function() {
 
     });
 
+    describe('given a burn-image-again state', function() {
```

Comment:
Notice that `SelectionStateService` doesn't know that "burn-image-again" means. In fact, it's completely isolated from the fact that the information it holds is used to burn something, therefore the title should be more generic, like "given the preserveImage option in clear()".